runelite-api/src/main/java/net/runelite/api/MenuAction.java

```diff
@@ -272,6 +272,10 @@
 	 * Menu action for configuring runelite overlays.
 	 */
 	RUNELITE_OVERLAY_CONFIG(1502),
+	/**
+	 * Menu action for triggering Emoji output to chat.
+	 */
+	RUNELITE_EMOJI_OUTPUT(1503),
```

Comment:
Don't add a new menu action type. Use `RUNELITE` instead and check the menu entry's text when clicked.